test/battle/ability/emergency_exit.c

```diff
@@ -36,3 +36,22 @@ SINGLE_BATTLE_TEST("Emergency Exit switches out when taking 50% max-hp damage af
         NOT ABILITY_POPUP(opponent, ABILITY_EMERGENCY_EXIT);
     }
 }
+
+SINGLE_BATTLE_TEST("xx Emergency Exit switches out when taking 50% max-hp damage after a restore hp hold effect was used")
+{
+    GIVEN {
+        PLAYER(SPECIES_WOBBUFFET)
+        OPPONENT(SPECIES_GOLISOPOD) { Ability(ABILITY_EMERGENCY_EXIT); MaxHP(263); HP(133); Item(ITEM_ORAN_BERRY); };
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN {
+            MOVE(player, MOVE_SUPER_FANG);
+            SEND_OUT(opponent, 1);
+        }
```

Comment:
Don't we generally keep each turn on a single line?

Reply:
fixed